rudder-agent/SOURCES/Makefile.in

```diff
@@ -31,7 +31,7 @@ zlib_RELEASE = 1.2.11
 zlib_SHA256 = c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1
 # Original URL: http://ftp.fr.debian.org/debian/pool/main/l/lmdb/lmdb_$(LMDB_RELEASE).orig.tar.xz
 lmdb_RELEASE = 0.9.24
-lmdb_SHA256 = 8e7200fdc9bb09b0cfad1d4eee8b20d539e43fe05c7ef9406247d64659c4a480
```

Comment:
why does this change?

Reply:
because it was forgotten when dependencies were upgraded, that why 7.0 dosen't build since then

Reply:
Ok :+1: